LayoutTests/imported/w3c/web-platform-tests/css/css-viewport/zoom/svg-computed-style.html

```diff
@@ -45,6 +49,14 @@
       "value": "-10px 20px 30px -40px",
       "otherValues": ["-20px 40px 60px -80px"]
     },
+    "min-height" : {
+      "value": "9px",
+      "otherValues": ["18px"]
+    },
+    "min-width" : {
+      "value": "9px",
+      "otherValues": ["18px"]
+    },
```

Comment:
```suggestion
    "min-height": {
      "value": "9px",
      "otherValues": ["18px"]
    },
    "min-width": {
      "value": "9px",
      "otherValues": ["18px"]
    },
```

Can you also add max as well?